metalog.js

```diff
@@ -287,6 +287,13 @@ class Logger extends events.EventEmitter {
   bind(application) {
     return new ApplicationLogger(this, application);
   }
+
+  // Checks whether passed `type` of logs is active.
+  //   type <string> logger type
+  // Returns: <boolean>
+  isActive(type) {
+    return this.toStdout[type] || this.toFile[type];
```

Comment:
A couple of nits:

- Maybe add a check for `this.active` and return `false` if it is `false`?
- The resulting value should probably be converted to Boolean before returning, so that the description above is correct. 

```suggestion
    if (!this.active) return false;
    return Boolean(this.toStdout[type] || this.toFile[type]);
```